easybuild/easyconfigs/r/RMBlast/RMBlast-2.2.28-linux64-Python-2.7.11.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'RMBlast'
+version = '2.2.28-linux64'
```

Comment:
I would make this `2.2.28`, and move the `-linux64` part to the `versionsuffix` (or maybe just remove it?)